toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/plan/TestPlanDaoService.java

```diff
@@ -12,75 +12,79 @@
 import com.google.inject.assistedinject.Assisted;
 import com.google.inject.name.Named;
 
+import io.toast.tk.dao.domain.impl.common.IServiceFactory;
 import io.toast.tk.dao.domain.impl.report.Campaign;
-import io.toast.tk.dao.domain.impl.report.Project;
+import io.toast.tk.dao.domain.impl.report.TestPlanImpl;
+import io.toast.tk.dao.domain.impl.repository.ProjectImpl;
+import io.toast.tk.dao.service.dao.access.repository.ProjectDaoService;
 import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
 import io.toast.tk.dao.service.dao.common.CommonMongoDaoService;
 import io.toast.tk.dao.service.init.DbStarter;
 
-public class ProjectDaoService extends AbstractMongoDaoService<Project> {
+public class TestPlanDaoService extends AbstractMongoDaoService<TestPlanImpl> {
 
-	public interface Factory {
-		
-		ProjectDaoService create(final @Assisted String dbName);
+	public interface Factory extends IServiceFactory<TestPlanDaoService>{
 	}
 
 	private final CampaignDaoService cDaoService;
+	private final ProjectDaoService pDaoService;
 
 	@Inject
-	public ProjectDaoService(
+	public TestPlanDaoService(
 		final DbStarter starter,
 		final CommonMongoDaoService cService,
 		final @Assisted String dbName,

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)